### ISSUE TITLE
Fatal error: Uncaught Error: Call to undefined function bp_nouveau()

### ISSUE THREAD

**Issue by tw2113** [WebDevStudios/BuddyPress-Registration-Options#213]
```
Fatal error: Uncaught Error: Call to undefined function bp_nouveau() in /bp-registration-options/includes/core.php:921
```

**Comment by tw2113** [comment]
Pending confirmation of working, but b8d36a5

**Comment by tw2113** [comment]
Fixed in 4.4.2
